cmd/manager/main.go

```diff
@@ -134,7 +133,8 @@ func main() {
 		os.Exit(1)
 	}
 
-	nsClient := client.NewNamespacedClient(mgr.GetClient(), namespace)
+	// nsClient := client.NewNamespacedClient(mgr.GetClient(), namespace)
```

Comment:
we can remove this line

Reply:
[8542b50](https://github.com/percona/percona-server-mysql-operator/pull/673/commits/8542b503ac935b67b810640eefb11154052b0f9a)